dune_api_scripts/queries/orderbook/order_rewards.sql

```diff
@@ -0,0 +1,34 @@
+with trade_hashes as (SELECT solver,
```

Comment:
This file is also copied from solver-rewards (except that the `order_uid` column is included in the final results)

cf.https://github.com/cowprotocol/solver-rewards/blob/main/queries/cow_rewards.sql

Reply:
If you wanna depend on other repo's code you can do it with: 
https://www.atlassian.com/git/tutorials/git-submodule#:~:text=A%20git%20submodule%20is%20a,the%20host%20repository%20is%20updated.

But maybe the best way would be publish a new repo called "dune-updater-template" that contains the logic to build a dune updater (maybe it should also go into your existing python dune library). And then we can use it in solver-rewards to build a docker for the new data-flow that updates the data in dune.

Reply:
> If you wanna depend on other repo's code you can do it with:

I don't really want to depend on the other repo because this query contains more data (the `order_uid`).

> But maybe the best way would be publish a new repo called "dune-updater-template" that contains the logic to build a dune updater (maybe it should also go into your existing python dune library). 

This I like a lot. I could move the `push_paginated_view` into the legacy API service.

I did also consider making a stand alone project/service just for this. If we do go this route, then we may also want to move out the retention updating and app_data... 

I am not exactly sure how we are going to update this from within the solver rewards although that would be nice. Can we execute docker images from within a python script? Was that what you had in mind? maybe we need docker-compose... its weird.

Reply:
Additionally, once this service goes live this query will not be needed anymore in solver rewards since that project only needs the aggregated sums. It will not depend on this table.  This view is merely for data availability purposes and will be linked to from the rewards dashboard.

Reply:
>Was that what you had in mind?

I was thinking that we could have a docker image of a basic updater, a sceleton that runs just any script specified in the docker-run command. And then in solver rewards, we can depend on this docker image and build a new one with it that contains the additional code. 
But I am really not sure whether we should go this route ... Let's discuss tomorrow in sycn

Reply:
Sure - although we don't actually need this table for the solver rewards - only for making the granular information available in the dashboard. So there two services are technically independent. However, if we did combine them, then we could use the Dune table for the payouts (but I think that is too risky). The payout needs to use the data coming directly from the orderbook (and not a user generated view).